### PR TITLE
[NEW] create a method 'create token'

### PR DESCRIPTION
**Pull request by ggazzo**
@RocketChat/core 

Allowed users (usually admins) can generate access tokens to another user.
Users can generate theirs tokens too.

### CODE REVIEW

File: packages/rocketchat-lib/server/methods/createToken.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Meteor.methods({
+	createToken({user}) {
```
Comment (graywolf336) [inline packages/rocketchat-lib/server/methods/createToken.js]: Change this parameter to only show which properties of the user you need. This way the clients calling this real-time method don't need to pass the full user object which in turn makes transmission over the wire faster.

**Comment by ggazzo** [comment]
@graywolf336 I think @rodrigok have a better answer. But I believe the case is a service/integration that uses the API, but we don't want to save passwords to get the token.

**Comment by graywolf336** [comment]
Okay. We've had a lot of people expecting the login via the rest api actually logging them into the website, which is why I was asking

**Comment by marceloschmidt** [comment]
@RocketChat/core this is a **huge** security flaw! How's this been approved? So an admin can now create a token for **any** user and login as them?

**Pull request merged**